### PR TITLE
CAS-7696: Turn off extended filename support if specified file exists

### PR DESCRIPTION
**Pull request by dmehring**
Before this change, importing uvfits files with certain names would fail because FITS extended filename support was enabled. Now we first check the file name, and if that file exists, this support is disabled to allow import of those files which would otherwise fail if such support was enabled.

### CODE REVIEW

File: fits/FITS/blockio.cc [hunk 1]
```diff
@@ -73,7 +74,14 @@ namespace casacore { //# NAMESPACE CASACORE - BEGIN
 	strcpy(m_filename,f);
 	// using cfitsio of NASA to open fits file for writting and reading.
 	int l_status, iomode;
-	l_status = 0; 
+	File myfile(m_filename);
+	if (myfile.exists() && myfile.isReadable()) {
```
Comment (juliantaylor) [inline fits/FITS/blockio.cc]: `isReadable` also checks existance, so the existance check is unnecessary (but also harmless)

**Comment by dmehring** [comment]
removed exists() check

**Comment by juliantaylor** [comment]
thanks

**Comment by juliantaylor** [comment]
for reference, an example filename is: `name+434` which is interpreted by cfitsio as open extension `434` of file `name`

**Pull request merged**